packages/common/index.ts

```diff
@@ -12,6 +12,17 @@ export type GetClubResponse = Club[];
 export type CreateClubParams = { name: string; rating: number };
 export type CreateClubResponse = Club;
 
+export type GetCourseResponse = {
+  name: string;
+  officeHours: Array<{
+    id: number;
+    title: string;
+    room: string;
+    startTime: Date;
+    endTime: Date;
+  }>;
```

Comment:
Would it be better to make this object type into an interface? That way the frontend can know what data type to expect inside the array.